server/channels/api4/content_flagging.go

```diff
@@ -598,3 +603,9 @@ func assignFlaggedPostReviewer(c *Context, w http.ResponseWriter, r *http.Reques
 	auditRec.Success()
 	writeOKResponse(w)
 }
+
+func checkForAllowedContentFlaggingPostTypes(c *Context, post *model.Post) {
+	if post.Type == model.PostTypeBurnOnRead {
```

Comment:
wondering is this is the only post type we should be filtering here, what about system messages?